pkg/constants/constants.go

```diff
@@ -10,3 +10,9 @@ const (
 	LvlDebug              // 4 - Debug-level logs, tricky logic areas
 	LvlTrace              // 5 - Trace-level logs, detailed troubleshooting context
 )
+
+// Service initialization constants
+const (
+	MetricsPath = "/metrics"
+	MetricsAddress = ":8080"
```

Comment:
should this go directly in the `metrics.go` file ? It's not going to be used any where else ?

Reply:
Should the address use only local interface `127.0.0.1` by default ? And the port is already used by cloudserver, can we pick another one that's not used by other components ?

Reply:
Yes you are right it will go in the metrics.go, the GO way is to keep it closer indeed if not being re-used.

To address your concerns:
	1.	Service Context: This service is not part of RING and will be deployed on customers’ Kubernetes clusters. Since it is deployed externally, it won’t conflict with any port numbers within RING.
	2.	Deployment Setup:
	•	The container is deployed as a Kubernetes pod, and the metrics route is exposed via a Kubernetes service with its own unique cluster IP.
	•	End-users can modify the exposed port via [deployment configurations or Helm charts](https://github.com/scality/cosi-driver/pull/72/files#diff-7cc8789e48e2353deaa6efa8224e6c65448c3da297d717784211324f6df0d821R34).
	3.	Port Conflict Analysis:
	•	Even without modifying the port, each Kubernetes service gets a unique IP.
	•	Since only one HTTP server (for metrics) runs within the pod, conflicts are highly unlikely within the pod.

Here’s an example for clarity:
```sh
✗ kubectl get svc --all-namespaces  
NAMESPACE                NAME                          TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                  AGE  
default                  kubernetes                    ClusterIP   10.96.0.1       <none>        443/TCP                  17h  
kube-system              kube-dns                      ClusterIP   10.96.0.10      <none>        53/UDP,53/TCP,9153/TCP   17h  
kubernetes-dashboard     dashboard-metrics-scraper     ClusterIP   10.103.11.103   <none>        8000/TCP                 17h  
kubernetes-dashboard     kubernetes-dashboard          ClusterIP   10.96.45.42     <none>        80/TCP                   17h  
scality-object-storage   scality-cosi-driver-metrics   ClusterIP   10.97.169.22    <none>        8080/TCP                 17h  
```